utils/hare-shutdown

```diff
@@ -20,190 +20,16 @@
 # :help: stop the cluster
 
 import logging
-import os
-import re
-import threading
-import argparse
-from queue import Queue
-from socket import gethostname
-from typing import Dict, List, NamedTuple
-
-from consul import Consul, ConsulException
-from requests.exceptions import RequestException
-from urllib3.exceptions import HTTPError
-from hax.exception import HAConsistencyException
+from utilitity import setup_logging, consul_is_active_at
+from utilitity import pcs_consul_is_active_at, processes_by_consul_svc_name
+from utilitity import get_kv, shutdown_sequence, stop_parallel
+from utilitity import is_fake_leader_name, ssh_prefix
+from consul import Consul
```

Comment:
Need to add line 27 and 28 to supress errors. Consul and repeat_if_fails  is not used here.